.rubocop.yml

```diff
@@ -3,3 +3,8 @@ inherit_from: .rubocop_todo.yml
 AllCops:
     NewCops: enable
 
+Metrics/BlockLength:
```

Comment:
I think both those metrics better to leave in `.rubocop_todo.yml` file since they both can be fixed.

Fixing them can take a lot of time, but they are fixable

I think general rule - disable or add specific values in `.rubocop.yml` file only in cases that you now fixing them is impossible, or fixing them broke something